bundler/spec/dependabot/bundler/file_updater_spec.rb

```diff
@@ -50,7 +50,9 @@
     )
   end
 
-  before { FileUtils.mkdir_p(tmp_path) }
+  before do
+    FileUtils.mkdir_p(tmp_path)
+  end
```

Comment:
What's the impetus for this change?

Reply:
revert back. Thanks for pointing out that.